common/cmake/litex.cmake

```diff
@@ -73,8 +74,8 @@ function(ADD_LITEX_TEST)
     OUTPUT ${LITEX_SOURCES}
     DEPENDS ${PYTHON3} ${DEPS}
     COMMAND
-      ${CMAKE_COMMAND} -E env PYTHON=${PYTHON3}
-      ${PYTHON3} ${GENERATE_SCRIPT} --board ${LITEX_BOARD} --builddir ${LITEX_DIR} --integrated-rom-size 0x10000 --uart-baudrate 1000000 ${FLAGS}
+    ${CMAKE_COMMAND} -E env PYTHON=${PYTHON3} LITEX_DIRECTORY=${LITEX_THIRD_PARTY}
```

Comment:
Indentation error.

One question also, why is `LITEX_DIRECTORY` needed? Is it used by litex internally?

Reply:
This was needed for generating the emulator and buildroot, but it has been dropped in this PR. Hence this is not needed anymore. Removed.